packages/astro/src/assets/utils/resolveImports.ts

```diff
@@ -0,0 +1,40 @@
+import { resolve } from 'node:path/posix';
```

Comment:
Could you move this file to a `node` folder? Either in this folder or at the root of `src/assets`. We somewhat try to do this pattern where possible to avoid someone importing something that imports Node stuff by accident. Some of the utils in this folder are exposed to users, so the distinction is useful

Reply:
I realised this is needed in the runtime, so I've removed all the node imports